modules/ocf/facts.d/dev-config

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash -eu
+if hostname | grep -qE '^dev-'; then
+    echo 'dev_config=true'
```

Comment:
`dev_host` might be a better name. i might even change this to not be a boolean, e.g. `environment=prod` or `environment=dev`